app/sidekiq/evss/disability_compensation_form/upload_bdd_instructions.rb

```diff
@@ -67,6 +77,21 @@ class UploadBddInstructions < Job
         }
       )
 
+      def self.api_upload_provider(submission)
```

Comment:
We need to do this in a class method so we can use the providers in both the job instance itself (i.e. in the `perform` behavior) and here in the `sidekiq_retries_exhausted` block, which fires after the job has died so we no longer have access to its instance methods.